File: java/src/automation-root/automation-extensions/src/main/java/com/automation/extensions/contracts/DriverParams.java
package com.automation.extensions.contracts;

import com.google.gson.annotations.Expose;

public class DriverParams {
    @Expose
    private String driver;

    @Expose
    private String binaries;

    public String getDriver() {
        return driver;
    }

    public DriverParams setDriver(String driver) {
        this.driver = driver;
        return this;
    }

    public String getBinaries() {
        return binaries;
    }

    public DriverParams setBinaries(String binaries) {
        this.binaries = binaries;
        return this;
    }
}

File: csharp/src/AutomationRoot/Automation.Extensions/Contracts/DriverParams.cs
using System.Runtime.Serialization;

namespace Automation.Extensions.Contracts
{
    [DataContract]
    public class DriverParams
    {
        [DataMember]
        public string Source { get; set; }

        [DataMember]
        public string Driver { get; set; }

        [DataMember]
        public string Binaries { get; set; }
    }
}
